tests/PKSim.Tests/IntegrationTests/IndividualParametersSameFormulaOrValueForAllSpeciesRepositorySpecs.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Utility.Container;
+using PKSim.Core.Model;
+using PKSim.Core.Repositories;
+using PKSim.Infrastructure.ORM.Repositories;
+
+namespace PKSim.IntegrationTests
+{
+   public abstract class concern_for_IndividualParametersSameFormulaOrValueForAllSpeciesRepository : ContextForIntegration<IIndividualParametersSameFormulaOrValueForAllSpeciesRepository>
+   {
+   }
+
+   public class When_retrieving_parameters_with_the_same_formula_or_value_for_all_species_from_the_repository : concern_for_IndividualParametersSameFormulaOrValueForAllSpeciesRepository
+   {
+      private IEnumerable<IndividualParameterSameFormulaOrValueForAllSpecies> _result;
+
+      protected override void Because()
+      {
+         _result = sut.All();
+      }
+
+      [Observation]
+      public void should_return_at_least_one_parameter()
+      {
+         _result.Count().ShouldBeGreaterThan(0);
+      }
+   }
+
+   public class When_testing_if_a_parameter_has_the_same_formula_or_value_for_all_species : concern_for_IndividualParametersSameFormulaOrValueForAllSpeciesRepository
+   {
+      private readonly IFlatContainerRepository _flatContainerRepository = IoC.Resolve<IFlatContainerRepository>();
+
+      private ParameterMetaData parameterMetaDataFor(string containerPath, string parameterName)
+      {
+         var containerId = _flatContainerRepository.ContainerFrom(containerPath).Id;
+
+         return new ParameterMetaData()
+         {
+            ContainerId = containerId,
+            ParameterName = parameterName
+         };
+      }
+
+      private (bool isSameFormula, bool isSameValue) isSameFormulaOrValue(string containerPath, string parameterName)
+      {
+         return sut.IsSameFormulaOrValue(parameterMetaDataFor(containerPath, parameterName));
+      }
+
+      private bool isSameFormula(string containerPath, string parameterName)
+      {
+         return sut.IsSameFormula(parameterMetaDataFor(containerPath, parameterName));
+      }
+
+      private bool isSameValue(string containerPath, string parameterName)
+      {
+         return sut.IsSameValue(parameterMetaDataFor(containerPath, parameterName));
+      }
+
+      protected static IEnumerable<object[]> TestData()
+      {
+         yield return new object[]
+         {
+            "Organism", "Weight", (isSameFormula:true, isSameValue:false)
+         };
+
+         yield return new object[]
+         {
+            "Organism", "Height", (isSameFormula:false, isSameValue:false)
+         };
+
+         yield return new object[]
+         {
+            "Organism|Bone", "Volume", (isSameFormula:false, isSameValue:false)
+         };
+
+         yield return new object[]
+         {
+            "Organism|Lumen|Duodenum", "Length", (isSameFormula:false, isSameValue:false)
+         };
```

Comment:
Lumen/Duodenum/Length is defined by formula for humans and by (different) values for animals
=>neither same formula nor same value